JS/02/table/style.scss

```
@@ -0,0 +1,71 @@
+input[type="checkbox"] {
+  outline: none;
+}
+.visi.visi {
+  opacity: 1;
+  visibility: visible;
+  animation: addUser 0.2s ease-in-out
+}
+.popup-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0,0,0,0.2);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  opacity: 0;
+  visibility: hidden;
+  z-index: 2;
+}
+.box-popup {
+  position: relative;
+  z-index: 3;
+  width: 500px;
+  background-color: #fff;
+  border-radius: 3px;
+  padding: 15px 20px;
+}
+.overlay-close {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  z-index: 2;
+  cursor: default;
+}
+.icon-close {
+  position: absolute;
+  top: 5px;
+  right: 10px;
+  font-size: 18px;
+  cursor: pointer;
+}
+.icon-close:hover {
+  color: red;
+}
+.d-inline-block.d-inline-block {
+  display: inline-block;
+}
+.box-table {
+  padding: 20px
+}
+.box-active {
+  text-align: right;
+  margin-bottom: 15px;
+}
+.box_active-table--save {
+  display: none;
+}
+.box_active-table--cancel {
+  display: none;
+}
```

should apply scss nested
```suggestion
.box-table {
   padding: 20px;
  
    .box-active {
    text-align: right;
    margin-bottom: 15px;
  }
}
```